### PR TITLE
Ratwood is dead

### PR DESCRIPTION
**Pull request by ikrarkjr**
Play on azure peak

PS: Enjoy having the specials and weighting be fucking dogshit because maintainers wouldnt merge this within 2 weeks or even reply to me lmfaooo

### CODE REVIEW

File: code/datums/special_traits/traits/traits.dm [hunk 1]
```diff
@@ -131,94 +147,97 @@
...
 /datum/special_trait/corn_fed/on_apply(mob/living/carbon/human/character, silent)
-	character.change_stat("constitution", 2)
+	character.change_stat("constitution", 6)
 	character.change_stat("intelligence", -2)
 
 /datum/special_trait/bookworm
 	name = "Bookworm"
 	greet_text = span_notice("I'm a fan of books and I enjoy reading them regularly.")
-	weight = 100
+	weight = 0
```
Comment (atomicplan666) [inline code/datums/special_traits/traits/traits.dm:159]: Ayo WHAT THE FUCK, I LOVE READING

File: code/datums/special_traits/traits/traits.dm [hunk 2]
```diff
@@ -289,33 +317,37 @@
 /datum/special_trait/gourmand
 	name = "Gourmand"
 	greet_text = span_notice("I can eat even the most spoiled, raw, or toxic food and water as if they were delicacies..")
-	weight = 100
+	weight = 200
 
 /datum/special_trait/gourmand/on_apply(mob/living/carbon/human/character, silent)
 	ADD_TRAIT(character, TRAIT_NASTY_EATER, "[type]")
+	character.change_stat("endurance", 4) // If you can endure drinking sewage...
```
Comment (Squird5) [inline code/datums/special_traits/traits/traits.dm:324]: Shouldn't this be Constitution rather than Endurance? Health, rather than Stamina.

Comment (ikrarkjr) [reply code/datums/special_traits/traits/traits.dm:324]: No, they aren't HEALTHY from consuming rotten trash. They have have the STAMINA to stomach it.

File: code/datums/special_traits/traits/traits.dm (hunk 1 above)
Comment (ikrarkjr) [reply code/datums/special_traits/traits/traits.dm:159]: I could just make it so this also gives you intelligence stat. ATM, it doesn't do anything at all.

File: code/datums/special_traits/traits/traits.dm (hunk 2 above)
Comment (john-pawson) [reply code/datums/special_traits/traits/traits.dm:324]: Noonononono stomaching it is definitely CON,END is things like enduring pain and being able to keep on running after a while without running out of breath, CON is things like not passing out from poisons and tranquilizers, eating food without throwing up and not getting knocked out after a couple of punches to the head.

**Comment by radarseas** [comment]
while you're here, could you correct night owl's description from "prefered noc over her sister" to, "prefered noc over his sister". Since lore-wise noc is refered to as a lord several times, including the in-built book "legend of the nitebeast".

**Comment by ikrarkjr** [comment]
> while you're here, could you correct night owl's description from "prefered noc over her sister" to, "prefered noc over his sister". Since lore-wise noc is refered to as a lord several times, including the in-built book "legend of the nitebeast".
> 
> _its always bugged me._

A lot of the descriptions have no punctuation, random capitalization, and are clearly low effort. They have almost all been rewritten now.

**Pull request closed without merging**